core/src/main/resources/schemas/xenc-schema.xsd

```diff
@@ -7,7 +17,7 @@
         elementFormDefault='qualified'>
 
   <import namespace='http://www.w3.org/2000/09/xmldsig#'
-          schemaLocation='xmldsig-core-schema.xsd'/>
+          schemaLocation='http://www.w3.org/TR/2002/REC-xmldsig-core-20020212/xmldsig-core-schema.xsd'/>
```

Comment:
Why are we changing local path to urls?

Reply:
For the same reason as above, the preamble prescribed at https://www.w3.org/TR/2002/REC-xmlenc-core-20021210/Overview.html#sec-Encryption-Syntax uses the remote URL. Again, this should not impact the local resolution of schemas provided by this PR, so we may omit this change as well.

This said, the currently used xenc-schema.xsd is quite different from the one at https://www.w3.org/TR/2002/REC-xmlenc-core-20021210/xenc-schema.xsd: for instance, single quotes are used instead of double quotes. I hope the other changes only involve whitespace, anyway I don't know if it's worth to replace the XSD currently bundled with java-saml with the official one at w3.org for safety, I leave the decision up to you.